src/apps/invitations/test_invite.py

```diff
@@ -962,3 +979,20 @@ async def test_shell_invite_cant_twice(self, client, session, shell_create_data,
         assert response.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
         message = response.json()["result"][0]["message"]
         assert message == RespondentInvitationExist.message
+
+    async def test_cant_create_invitation_with_same_secret_id_as_shell_account(
+        self,
+        client,
+        session,
+        applet_one: AppletFull,
+        applet_one_shell_account: Subject,
+        tom: User,
+        invitation_respondent_data,
+    ):
+        await client.login(self.login_url, tom.email_encrypted, "Test1234!")
+        invitation_respondent_data.secret_user_id = applet_one_shell_account.secret_user_id
+        response = await client.post(
+            self.invite_respondent_url.format(applet_id=str(applet_one.id)),
+            invitation_respondent_data.dict(),
+        )
+        assert response.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
```

Comment:
It will be better to check error message in response to be sure that we have 422 status code because of the same secret_id